packages/wallet/src/Wallets/BaseWallet.ts

```diff
@@ -878,15 +878,33 @@ export class BaseWallet implements ObservableWallet {
     return isEmpty ? [knownAddresses[0]] : [];
   }
 
-  /** Update the witness of a transaction with witness provided by this wallet */
-  async updateWitness({ tx, sender }: UpdateWitnessProps): Promise<Cardano.Tx> {
-    return this.finalizeTx({
-      auxiliaryData: tx.auxiliaryData,
+  async addSignatures({ tx, sender }: AddSignaturesProps): Promise<Serialization.TxCBOR> {
```

Comment:
We should add some test(s) in `methods.test.ts`

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1411/commits/892c1e9ae82302eb4f07cfc033c3867eaee11c5a